manifests/l/Lego/MindstormsEduNXTSoftware/2.1/Lego.MindstormsEduNXTSoftware.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Lego.MindstormsEduNXTSoftware
+PackageVersion: 2.1

```

This must match control panel; When updating this field, you will also need to update the name of the folder the manifests are in
```suggestion
PackageVersion: 2.1.126.0
```